.github/workflows/ci.yml

```diff
@@ -137,6 +141,9 @@ jobs:
     runs-on: ${{ vars.RUNNER_MEDIUM || 'ubuntu-latest-m' }}
     timeout-minutes: ${{ vars.TIMEOUT_MINUTES_LONG && fromJSON(vars.TIMEOUT_MINUTES_LONG) || 15 }}
 
+    env:
```

Comment:
Used to explicitly set NODE_VERSION between runs. Else most runs are using `18.20.2`, while `Unit Tests (18)` is using `^18`